website/docs/d/synthetics_monitor_location.html.markdown

```diff
@@ -0,0 +1,42 @@
+---
+layout: "newrelic"
+page_title: "New Relic: newrelic_synthetics_monitor_location"
+sidebar_current: "docs-newrelic-datasource-synthetics-monitor-location"
+description: |-
+  Grabs a Synthetics monitor location by label.
+---
+
+# Data Source: newrelic\_synthetics\_monitor
+
+Use this data source to get information about a specific Synthetics monitor locations in New Relic that already exist. This can be used to set up a Synthetics alert condition.
```

Comment:
nit: can we swap "locations" for "location" here, and remove "This can be used to set up a Synthetics alert condition"?